docs/integrations/llms/anthropic.md

```diff
@@ -103,3 +103,23 @@ Shows up like this in Logfire:
   ![Logfire Anthropic Streaming](../../images/logfire-screenshot-anthropic-stream.png){ width="500" }
   <figcaption>Anthropic streaming response</figcaption>
 </figure>
+
+# Amazon Bedrock
+You can also log Anthropic LLM calls to Amazon Bedrock using the `AmazonBedrock` and `AsyncAmazonBedrock` clients.
+
+To use this integration, you'll need the AWS SDK dependencies:
+- `boto3`
+- `botocore`
+
+You can install them along with other required packages using:
+```pip install -U logfire[bedrock]```
```

Comment:
this should probably be using the special extra installation block that other pages use.

Reply:
Removed as discussed